quaint/src/connector/connection_info.rs

```diff
@@ -34,7 +36,10 @@ pub enum ConnectionInfo {
         db_name: String,
     },
     #[cfg(feature = "sqlite")]
-    InMemorySqlite { db_name: String },
+    InMemorySqlite {
+        db_name: String,
+    },
+    External(ExternalConnectionInfo),
```

Comment:
Here we expanding `ConnectionInfo` with `External` option that basically means any queryable that is not one of the built in ones. It was needed, because it's not trivial to construct other options without a proper URL - most of the fields would need to be made optional.